packages/terra-collapsible-menu-view/tests/jest/CollapsibleMenuViewDivider.test.jsx

```diff
@@ -16,6 +16,17 @@ describe('CollapsibleMenuViewDivider', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('should render a chevron right icon', () => {
+    const wrapper = shallowWithIntl(<CollapsibleMenuViewDivider useChevronRightIcon />);
```

Comment:
nit: Can we add assertions for these tests? Like for this one actually checking for the existence of the icon programmatically